action.yml

```diff
@@ -4,6 +4,7 @@ inputs:
   actions_list:
     description: List of private actions to checkout. Must be a JSON array and each entry must mutch the format owner/repo@ref
     required: true
+    default: '[]'
```

Comment:
@gwatts maybe we can have the `required` here as `false` since we now have a default value. I was thinking... the original purpose of this action was to clone other **private actions** I needed in my workflows (corporate code in private repositories). We are evolving the code and with your contribution we can now use this custom action to setup the git environment to get access to any private repo (not only actions), which is awesome. The action's name (**private-actions-checkout**) will be a little outdated! Maybe a better name would be _private-repos-checkout_. How confortable are you with this naming conflict? Please share your thoughts!